.github/workflows/pr_build.yml

```diff
@@ -79,33 +79,12 @@ jobs:
         if: ${{ matrix.java == '8' }} # Gradle example can only be built on Java 8
         working-directory: examples/powertools-examples-core/kotlin
         run: ./gradlew build
-      - name: Setup Terraform
```

Comment:
Hey @jasoniharris does this PR  include the contents of #1533?

Reply:
Yes - My mistake - Just pushed to revert this to current main.